lib/ssh/src/ssh_sftp.erl

```diff
@@ -865,7 +865,12 @@ do_handle_reply(#state{xf = Xf} = State,
     case Xf#ssh_xfer.vsn of
 	undefined ->
 	    ok;
-	From ->
+	{wait, From, TRef} ->
+	    if is_reference(TRef) ->
```

Comment:
Because I wanted it to be clear(er) that the reply occurs regardless of TRef

874:    ssh_channel:reply(From, ok)

But I'm not style-conscious in any way - if you feel the readability is
improved with two patterns then please go ahead and amend it.

/Simon

On 16 April 2015 at 16:37, Pierre Fenoll notifications-at-github.com
|github/Allow| <0dq04cosrt@sneakemail.com> wrote:

> In lib/ssh/src/ssh_sftp.erl
> <https://github.com/erlang/otp/pull/682#discussion_r28559862>:
>
> > @@ -865,7 +865,12 @@ do_handle_reply(#state{xf = Xf} = State,
> >      case Xf#ssh_xfer.vsn of
> >  	undefined ->
> >  	    ok;
> > -	From ->
> > +	{wait, From, TRef} ->
> > +	    if is_reference(TRef) ->
>
> Why don't you just use a guard here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/erlang/otp/pull/682/files#r28559862>.
>